AMAZON-INSTRUCTIONS.md

```diff
@@ -0,0 +1,33 @@
+## Instructions
+
+Add a new dependency to the build.gradle apart from the regular `purchases` dependency. This new dependency has the classes needed to use Amazon IAP:
+
+```groovy
+dependencies {
+    implementation 'com.revenuecat.purchases:purchases:5.0.0-amazon.beta.1'
+    implementation 'com.revenuecat.purchases:purchases-store-amazon:5.0.0-amazon.beta.1'
+}
+```
+
+Configure the SDK using your **RevenueCat API key specific for Amazon**:
+
+```kotlin
+Purchases.configure(AmazonConfiguration.Builder(this, "api_key").build())
```

Comment:
we could really drive home the point that the amazon key is different by making the placeholders "amzn_api_key" and "googl_api_key" or whatever the prefixes in the dashboard are?

Reply:
Good point. I changed it to `amzn_api_key`. The official docs should also have a good explanation of this with screenshots and stuff so hopefully we are good!